build/wrapper.cc

```diff
@@ -200,10 +208,15 @@ std::shared_ptr<RayObject> get(const ObjectID object_id, int64_t timeout_ms) {
     std::vector<ObjectID> get_obj_ids = {object_id};
     auto status = worker.Get(get_obj_ids, timeout_ms, &objects);
     if (!status.ok()) {
-        return nullptr;
+        throw std::runtime_error(status.ToString());
+    }
+
+    if (objects.size() != 1) {
+        auto msg = "Size of RayObjects is " + std::to_string(objects.size());
+        auto st = Status::Invalid(msg);
+        throw std::runtime_error(st.ToString());
     }
```

Comment:
This is an internal error as the caller can only specify one object to get. As there is no "InternalError" I think "UnknownError" probably fits best:
```suggestion
    if (objects.size() != 1) {
        auto msg = "Requested a single object but instead found " + std::to_string(objects.size()) + " objects.";
        auto st = Status::UnknownError(msg);
        throw std::runtime_error(st.ToString());
    }
```